packages/component/src/hooks/internal/styleToEmotionObject.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { createContext, createRef, type ReactNode, type RefObject, useCallback, useContext } from 'react';
+import { useRefFrom } from 'use-ref-from';
+import { hooks } from 'botframework-webchat-api';
+import useEmotion from './useEmotion';
+
+const { useStyleOptions } = hooks;
+
+const styleToEmotionObjectContext = createContext<RefObject<(...args: any[]) => string>>(createRef());
+
+export const StyleToEmotionObjectProvider = ({ children, nonce }: Readonly<{ children: ReactNode; nonce: string }>) => {
```

Comment:
Could we turn this into our `<XXXProvider>` pattern? Like, the `<ActivityKeyerComposer>`. But I know the boilerplate code could be improved if we set up a standard pattern.

Reply:
Maybe a barrel file (`index.ts`) for the `<XXXProvider>` pattern so we have a single entry point. I forgot what come up with our last discussion about this.